.github/workflows/e2e-tests.yaml

```diff
@@ -49,6 +49,45 @@ jobs:
         # use for a container registry.
         registry: 'false'
 
+    - name: Read KinD network subnet
+      id: kind-subnet
+      run: |
+        kind_subnet_prefix="$(\
+          docker network inspect kind -f '{{ (index .IPAM.Config 0).Subnet }}' | cut -d'.' -f'1,2'\
+        )"
+
+        echo "Subnet prefix of 'kind' network: ${kind_subnet_prefix}"
+        echo "::set-output name=prefix::${kind_subnet_prefix}"
+
+    - name: MetalLB load-balancer
+      # Based on https://kind.sigs.k8s.io/docs/user/loadbalancer/
+      run: |
+        kubectl apply -f https://raw.githubusercontent.com/metallb/metallb/v0.11.0/manifests/namespace.yaml
```

Comment:
Should we migrate these manifests to our own repo? or is it safe to assume that these will stay in place?

Reply:
I think it's fine, this points to a tagged version (immutable!), and if MetalLB disappears having the manifests cached won't bring the actual app back.

We also install Knative from remote manifests (also tagged version).